inc/saibuffer.h

```diff
@@ -349,9 +365,9 @@ typedef enum _sai_buffer_profile_attr_t
      * If not set, use #SAI_BUFFER_POOL_ATTR_TH_MODE.
      * (default to value set in #SAI_BUFFER_POOL_ATTR_TH_MODE)
```

Comment:
I mean the default value should be SAI_BUFFER_PROFILE_THRESHOLD_MODE_INHERIT_BUFFER_POOL_MODE